CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+# SEED Version 2.12.2
+
+Date Range: 09/28/21 - 11/04/21
```

Comment:
Are these really all the closed tickets for 2.12.2? Seems like a lot! Happy if it is!

Reply:
Yeah, I think there might be some duplicates from the previous release... looking into that now...